okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -401,6 +436,7 @@ class Cookie private constructor(
       var httpOnly = false
       var hostOnly = true
       var persistent = false
+      var sameSite: String? = null
```

Comment:
default to `"lax"` here so we don’t need to fix it later?

Reply:
Updated, though I'm concerned the default may change in the future and we'll be providing an incorrect default.

Reply:
Yeah my bad. Because we’re going to differentiate between absent and Lax I think we should make absent the default.